GradedModulePresentationsForCAP/gap/GradedModulePresentations.gi

```diff
@@ -926,6 +954,54 @@ InstallGlobalFunction( ADD_GRADED_IDENTITY,
     
 end );
 
+##
+InstallGlobalFunction( ADD_GRADED_LIFT_ALONG_MONOMORPHISM,
+
+  function( category )
+    
+    AddLiftAlongMonomorphism( category,
+
+      function( iota, tau )
+        local lift;
+
+        lift := LiftAlongMonomorphism( UnderlyingPresentationMorphism( iota ), UnderlyingPresentationMorphism( tau ) );
+        
+        if lift = fail then
```

Comment:
`LiftAlongMonomorphism` and `ColiftAlongEpimorphism` should not return `fail`. They can throw an error, but maybe the garbage-in-garbage-out principle applies here, and we should assume that if they are called, the lift exists.

Reply:
Yes, you are right. The `fail` case can simply be deleted.

Reply:
Shouldn't the fail case throw a meaningful error? Or do we just rely on CAP giving some sensible error message? I would actually prefer the latter.

Reply:
If there is no graded lift along monomorphism then there also will be no non-graded lift along monomorphism (provided the input consists of graded morphisms).

Thus, an error would be thrown by the underlying non-graded case.

Reply:
Perfect, Then I will provide a patch for this PR.